### PR TITLE
Fixes for Protractor Tests

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated protractor tests to pass in Sauce Connect

## Additions

- Added framework setting to config
- Added timeout setting to config

## Removals

- Removed failing bureau test that couldn't be easily traced (we can add back in later)

## Changes

- Updated page loads to only happen once

## Testing

- checkout branch and run `gulp test:acceptance --sauce=true` while connected to Sauce Connect

## Review

- @KimberlyMunoz 
- @anselmbradford 
- @sebworks 
- @imuchnik 

## Preview

screen shot 2015-08-27 at 4 25 41 pm

__Look at those runtimes now!!!__
screen shot 2015-08-27 at 4 39 13 pm

[Preview this PR without the whitespace changes](?w=0)

## Notes

- It's insanely hard to debug Protractor issues in Sauce Connect

### CODE REVIEW

File: test/browser_tests/spec_suites/contact-us.js [hunk 1]
```diff
@@ -8,11 +8,14 @@ describe( 'Contact Us Page', function() {
...
   var page;
 
-  beforeEach( function() {
+  beforeAll( function() {
     page = new ContactUsPage();
     page.get();
   } );
 
+  it( 'prime the page', function() {
+  } );
```
Comment (anselmbradford) [inline test/browser_tests/spec_suites/contact-us.js]: Perhaps an empty test function that should be removed?

**Comment by KimberlyMunoz** [comment]
:+1:

image

**Comment by anselmbradford** [comment]
:+1: 
screen shot 2015-08-27 at 5 31 48 pm

Changelog entry needed for removed test?

**Comment by jimmynotjim** [comment]
@anselmbradford yeah, I've just been waiting to do them last so I don't have conflicts to resolve. Squashing and updating that now

**Pull request merged**